common/autoware_universe_utils/src/math/trigonometry.cpp

```diff
@@ -72,4 +72,57 @@ std::pair<float, float> sin_and_cos(float radian)
   }
 }
 
+// This code is modified from part of OpenCV project
```

Comment:
grammar

Reply:
Thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/8380/commits/57824efce82fcc9c1116d0d70d87600cff642b07